bin/seeds

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
 script/deps
 docker-compose run app mix ecto.setup
-script/prune
+cd "$(dirname "$0")"
+source prune
```

Comment:
might be time to think about combining all of these scripts that do different things into one script that does a bunch of things.

Reply:
Great idea!